src/components/shared/formItems/ProjectReserved.tsx

```diff
@@ -69,6 +91,16 @@ export default function ProjectReserved({
             {shouldRenderToggle ? (
               <SwitchHeading checked={Boolean(checked)} onChange={onToggled}>
                 <Trans>Reserved rate</Trans>
+                {!Boolean(checked) && (
+                  <span
+                    style={{
+                      color: colors.text.tertiary,
+                      marginLeft: 15,
+                    }}
+                  >
+                    (0%)
```

Comment:
**suggestion** can we source this from `defaultFundingCycleMetadata.reservedRate`? this way we create a single source of truth